request_handler.py

```diff
@@ -72,8 +73,19 @@ def do_GET(self):
         self.wfile.write(json.dumps(response).encode())
 
     def do_POST(self):
-        # TODO
-        pass
+        """Handles POST requests to the server
+        """
+        self._set_headers(201)
+        content_len = int(self.headers.get('content-length', 0))
+        post_body = self.rfile.read(content_len)
+        post_body = json.loads(post_body)
+        (resource, _) = self.parse_url(self.path)
+
+        new_comment = self.parse_url(self.path)
+
+        if resource == "comments":
+            new_comment = create_comment(post_body)
+            self.wfile.write(json.dumps(f"{new_comment}").encode())
```

Comment:
The f-string code needs to be removed here in order for the dictionary-within-a-list to return properly.